v3/src/v2/codap-v2-types.ts

```diff
@@ -128,6 +128,13 @@ export interface ICodapV2TableStorage extends ICodapV2BaseComponentStorage {
 export interface ICodapV2PlotStorage {
   verticalAxisIsY2: boolean
   adornments: Record<string, any>
+  areSquaresVisible: boolean
+  isLSRLVisible: boolean
+  movableLineStorage: Record<string, any>
+  movablePointStorage: Record<string, any>
+  multipleLSRLsStorage: Record<string, any>
+  multipleMovableValues: Record<string, any>
```

Comment:
Could try to type these better at some point, but I don't know how important it is, so it doesn't need to hold up the PR.

Reply:
@kswenson I'm not sure it's important either, but it seemed easy enough to do. So I went ahead and did that. I also added support for Connecting Lines. That also seemed pretty straightforward, but I'm going to request a follow-up review just in case.